samples/client/petstore/typescript-fetch/tests/default/tsconfig.json

```diff
@@ -5,12 +5,15 @@
         "noImplicitAny": true,
         "sourceMap": false,
         "outDir": "dist",
-        "rootDir": "."
+        "types": [
+            "mocha"
+        ],
+        "lib": [
+            "es6",
+            "dom"
+        ]
     },
     "exclude": [
-        "node_modules",
-        "typings/browser",
-        "typings/main",
-        "typings/main.d.ts"
+        "node_modules"
     ]
 }
```

Comment:
switch use type definition from `typings` to npm `@types/`, typings is [Deprecation](https://github.com/typings/typings#deprecation-notice-regarding-typescript20)

add node_modules exclude folder to fast typescript fast compiler